### PR TITLE
darts-shared-infrastructure

### PR DESCRIPTION
**Pull request by scott-robertson1**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: apps/darts-modernisation/darts-shared-infrastructure/demo.yaml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+apiVersion: helm.toolkit.fluxcd.io/v2beta1
+kind: HelmRelease
+metadata:
+  name: darts-shared-infrastructure
+  namespace: darts-modernisation
+spec:
+  releaseName: darts-shared-infrastructure
+  values:
+    java:
```
Comment (thomast1906) [inline apps/darts-modernisation/darts-shared-infrastructure/demo.yaml:9]: Are you missing specific demo java values ?

File: apps/darts-modernisation/darts-shared-infrastructure/test.yaml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+apiVersion: helm.toolkit.fluxcd.io/v2beta1
+kind: HelmRelease
+metadata:
+  name: darts-shared-infrastructure
+  namespace: darts-modernisation
+spec:
+  releaseName: darts-shared-infrastructure
+  values:
+    java:
```
Comment (thomast1906) [inline apps/darts-modernisation/darts-shared-infrastructure/test.yaml:9]: Are you missing specific demo java values ?

File: apps/darts-modernisation/demo/base/kustomization.yaml [hunk 3]
```diff
@@ -6,4 +6,7 @@ namespace: darts-modernisation
 patchesStrategicMerge:
   - ../../darts-api/demo.yaml
   - ../../darts-portal/demo.yaml
-  - ../../darts-gateway/demo.yaml
\ No newline at end of file
+  - ../../darts-gateway/demo.yaml
+  - ../../darts-shared-infrastructure/demo.yaml
+
+
```
Comment (thomast1906) [inline apps/darts-modernisation/demo/base/kustomization.yaml:12]: too much spacing- CI should fail

File: apps/darts-modernisation/stg/base/kustomization.yaml [hunk 4]
```diff
@@ -6,4 +6,4 @@ namespace: darts-modernisation
 patchesStrategicMerge:
   - ../../darts-api/stg.yaml
   - ../../darts-portal/stg.yaml
-  - ../../darts-gateway/stg.yaml
+  - ../../darts-shared-infrastructure/stg.yaml
```
Comment (thomast1906) [inline apps/darts-modernisation/stg/base/kustomization.yaml:9]: is that correct? renaming?

**Review by thomast1906 — CHANGES_REQUESTED** [review]
I think a few changes required, jhave left comments

File: apps/darts-modernisation/stg/base/kustomization.yaml (hunk 4 above)
Comment (scott-robertson1) [reply apps/darts-modernisation/stg/base/kustomization.yaml:10]: should be resolved now

File: apps/darts-modernisation/darts-shared-infrastructure/darts-shared-infrastructure.yaml [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+      chart: ./stable/darts-shared-infrastructure
+      sourceRef:
+        kind: GitRepository
+        name: hmcts-charts
+        namespace: flux-system
+      interval: 1m
+  values:
+    java:
+      image: sdshmctspublic.azurecr.io/darts/gateway:prod-6a10560-20230425114654 # {"$imagepolicy": "flux-system:darts-gateway"}
+      disableTraefikTls: true
```
Comment (thomast1906) [inline apps/darts-modernisation/darts-shared-infrastructure/darts-shared-infrastructure.yaml:19]: Dont think this value is needed

**Pull request closed without merging**